src/Admin/AdminConsole/Controllers/OrganizationsController.cs

```diff
@@ -255,6 +255,12 @@ public async Task<IActionResult> Edit(Guid id, OrganizationEditModel model)
             Seats = organization.Seats
         };
 
+        if (model.PlanType == PlanType.Free && model.Seats > 2)
+        {
+            TempData["Error"] = "Organizations with more than 2 seats cannot be downgraded to the Free plan";
+            return RedirectToAction("Edit", new { id });
+        }
```

Comment:
We've hit issues here before where existing orgs cannot be edited (for an unrelated reason) because it trips new validation code. For example, if we have any orgs today on the Free plan with more than 2 seats, CS will no longer be able to edit anything about them because they'll hit this error. (Obviously we _shouldn't_ have that but I can't guarantee it.)

I recommend only enforcing this if the plan has changed, e.g. `if (organization.PlanType != PlanType.Free && model.PlanType == PlanType.Free && model.Seats > 2)`.